src/borg/testsuite/archiver.py

```diff
@@ -57,6 +57,9 @@ def exec_cmd(*args, archiver=None, fork=False, exe=None, **kw):
         except subprocess.CalledProcessError as e:
             output = e.output
             ret = e.returncode
+        except SystemExit as e:
+            output = ''
+            ret = e.code
```

Comment:
the tests for bad filters wouln't work as argparse raises `SystemExit` which would abort pytest.